### PR TITLE
Support Platform Access Token

### PR DESCRIPTION
**Pull request by RobiNino**
- [x] All [tests](https://github.com/jfrog/jfrog-cli-core#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli-core/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----
Depends on https://github.com/jfrog/jfrog-client-go/pull/822

### CODE REVIEW

File: general/token/accesstokencreate.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
+package generic
```
Comment (yahavi) [inline general/token/accesstokencreate.go:1]:
```suggestion
package login
```

Comment (yahavi) [inline general/token/accesstokencreate.go:1]: Let's rename this file to "createaccesstoken"

File: general/token/accesstokencreate.go [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+	content, err := json.Marshal(*atc.response)
+	return content, errorutils.CheckError(err)
+}
+
+func (atc *AccessTokenCreateCommand) ServerDetails() (*config.ServerDetails, error) {
+	return atc.serverDetails, nil
+}
+
+func (atc *AccessTokenCreateCommand) CommandName() string {
+	return "access_token_create"
```
Comment (yahavi) [inline general/token/accesstokencreate.go:106]:
```suggestion
	return "create_access_token"
```

**Review by yahavi — CHANGES_REQUESTED** [review]

File: general/token/accesstokencreate.go (hunk 1 above)
Comment (RobiNino) [reply general/token/accesstokencreate.go:1]: Not part of the login mechanism. Changed to `token`

Comment (RobiNino) [reply general/token/accesstokencreate.go:1]: I think we should follow the convention where the file is named after the command it adds.

**Pull request merged**